### ISSUE TITLE
Failing BASIS system test after recent LoadEventNexus changes

### ISSUE THREAD

**Issue by martyngigg** [mantidproject/mantid#22929]
The [BASISAutoReduction system test](http://builds.mantidproject.org/view/Release%20Pipeline/job/release_systemtests-ubuntu-16.04/82/) is failing after #22902 was merged. The pull request was merged as it was thought that the only failing tests were ICAT tests.

The following script shows that the each value of the `event_index` field in the `SystemTest/BSS_13387_event.nxs` is the maximum value of a `uint_64`:
```python
from h5py import File
from pprint import pprint

filename = 'ExternalData/Testing/Data/SystemTest/BSS_13387_event.nxs'
nxsfile = File(filename)
entry = nxsfile['entry']
bank1_events = entry['bank1_events']
bank1_event_index = bank1_events['event_index']
pprint(bank1_event_index[:])
```
Output:
```
array([18446744073709551615, 18446744073709551615, 18446744073709551615,
       ..., 18446744073709551615, 18446744073709551615,
       18446744073709551615], dtype=uint64)
```

causing the current loading code to skip loading both banks.

**Comment by martyngigg** [comment]
As all pull requests onto `release-next` will be failing I have temporarily disabled the tests that are using this file until they can be updated.

**Comment by jmborr** [comment]
[BASISAutoReduction system test](http://builds.mantidproject.org/view/Release%20Pipeline/job/release_systemtests-ubuntu-16.04/82/) is no longer relevant to current BASIS reduction workflow. It is a legacy test. I will remove the test and the `md5` file it uses.
- ./Testing/Data/SystemTest/BSS_13387_event.nxs.md5
- ./Testing/Data/SystemTest/BSS_13387_event.nxs.expected.md5
